workflow/controller/node_counters.go

```
@@ -50,6 +54,14 @@ func (woc *wfOperationCtx) getUnsuccessfulChildren(boundaryID string) int64 {
 	return woc.countNodes(getUnsuccessfulChildrenCounter(boundaryID))
 }
 
+func (woc *wfOperationCtx) nodePodExist(node wfv1.NodeStatus) bool {
+	_, podExist, err := woc.controller.podInformer.GetIndexer().GetByKey(fmt.Sprintf("%s/%s", woc.wf.Namespace, node.ID))
+	if err != nil {
```

suggestion - ignore this error completely - it never seems to happen